app/build.gradle.kts

```diff
@@ -147,6 +147,7 @@ dependencies {
     implementation(projects.shared.base)
     implementation(projects.shared.data.core)
     implementation(projects.shared.domain)
+    implementation(projects.shared.ui.core)
```

Comment:
Do we need this dependency?

Reply:
Yes, unless we change the implementation

Reply:
Got it, let's revert all changes in `RootActivity`. The entire implementation can be encapsulated in the settings module